bot/strings.py

```diff
@@ -19,6 +19,9 @@ class Strings:
     # Function : fun.ping()
     PING = "Pong ! :ping_pong:"
 
+    # Function : fun.ping2()
+    PING2 = "Pong <@199621995913150464> ! :medhiblond: :ping_pong:"
```

Comment:
This will not work as :medhiblond: is a custom emoji. You can resolve this problem using https://discordpy.readthedocs.io/en/latest/faq.html#how-can-i-add-a-reaction-to-a-message